src/components/home-container/styles.js

```diff
@@ -1,3 +1,21 @@
-export const homeContianer = {
-    display: "flex"
-}
\ No newline at end of file
+export const homeContainer = {
+    alignItems: "center",
+    backgroundColor: "#333",
```

Comment:
lets change `backgroundColor` to `colors.violet300` and `color` to `colors.white` (will also need to import `colors` at the top of this file)